CONTRIB.md

```diff
@@ -49,17 +49,35 @@ $ docker exec -it openoversight-web-1 bash
 
 Once you're done, `make stop` and `make clean` to stop and remove the containers respectively.
 
-## Gmail Requirements
-**NOTE:** If you are running on dev and do not currently have a `service_account_key.json` file, create one and leave it empty. The email client will then default to an empty object and simulate emails in the logs.
+## Setting Up Email
+**NOTE:** If you are running on dev and do not currently have a `service_account_key.json` file, create one and leave it empty. This file is required by docker compose.
```

Comment:
This SHOULD be done automatically. You can remove this line because of this change: